app/javascript/components/search/controls/download/DownloadCommand.js

```diff
@@ -109,7 +109,7 @@ function getDownloadCommand(authCode, downloadId) {
   // instead of the hostname
   const downloadCommand = (
     `${curlExec} "${url}" -${curlSecureFlag}o cfg.txt; ` +
-    `${curlExec} -K cfg.txt && rm cfg.txt` // Removes only if curl succeeds
+    `${curlExec} -K cfg.txt; rm cfg.txt` // Removes only if curl succeeds
```

Comment:
BLOCKER: this changes the behavior, now we will delete cfg.txt regardless of success/fail.  I prefer the old behavior, since it makes troubleshooting much easier, but it's a bummer it's not supported in powershell.  Should we just tell users to use a cmd.exe shell instead?  Supporting two different windows download shells is going to be tricky.  

In any case, at the very least this needs the comment updated

Reply:
I agree that supporting multiple shells is not ideal.  After a lot of searching/testing, it appears we can do the following for PowerShell:

```
curl.exe -K cfg.txt ; if ($?) { rm cfg.txt }
```

I tested this and it worked.  I also did a whole bulk download request where I replaced `cfg.txt` in the middle command with `foo.txt`, and it correctly errored out and left `cfg.txt` in place.  I'm going to make this update then validate locally that the commands are generating properly.